script.js

```diff
@@ -49,4 +49,24 @@ closeButton.addEventListener('click', () => {
   const blurry = document.querySelector('.back');
   popUp.classList.add('popup1');
   blurry.style = 'display : none';
-});
\ No newline at end of file
+});
+
+const form = document.querySelector('.form');
+const email = document.querySelector('#email');
+const error = document.querySelector('.error');
+
+function checkEmail(Diana) {
+  if (Diana.match(/^([a-z0-9_])+(([a-z0-9])+\.)+([a-z0-9]{2,4})+$/)) {
```

Comment:
- [ ] Your regex is not recognizing a valid email and this is preventing the user to submit the form. Please change this method, you could compare the email with itself in lowercase using the `toLowerCase()` method.
 - [ ] Kindly change the attribute you used in this function. It is best practice to use argument names that mean what they do, like `emailAddress'.